hack/run_test

```diff
@@ -350,6 +352,43 @@ function setup_replication_cluster() {
   test_slave_visibility $cid_suffix
 }
 
+function run_master_restart_test() {
+  echo "Testing failed master restart"
+  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
+  local cid_suffix="mrestart"
+  local table_name="t1"
+
+  local volume_dir
+  volume_dir=`mktemp -d --tmpdir pg-testdata.XXXXX`
+  chmod a+rwx ${volume_dir}
+  master_args="-v ${volume_dir}:/var/lib/pgsql/data:Z"
+
+  # Setup the cluster
+  local slave_num=2
+  setup_replication_cluster $slave_num
```

Comment:
It then just looks weird (or complicated), because you are overwriting the `slave_num` from `run_master_restart_test` with another local variable (named exactly the same) ...  But the local variable from `run_master_...` is available in called `setup_replication_cluster` function, too.   Also, the local variable is not needed:
https://github.com/praiskup/openshift-postgresql/commit/c1b99705a2